blog/_posts/2021-07-09-FastAI-tabular-data.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Working with Tabular Data in FastAI.jl
+author: Manikya Bardhan
+layout: blog
+---
+
+[FastAI.jl](https://github.com/FluxML/FastAI.jl) is a package inspired by [fastai](https://github.com/fastai/fastai), and it's goal is to easily enable creating state-of-the-art models.
+
+This blog post shows how to get started on working with tabular data using FastAI.jl and related packages. The work being presented here was done as a part of [GSoC'21](https://summerofcode.withgoogle.com/projects/#5088642453733376) under the mentorship of Kyle Daruwalla, Brian Chen and Lorenz Ohly.
+
+## Loading the data in a container
+
+To start working, we'll have to take our tabular data and load it in such that it supports the interface defined by [Tables.jl](https://tables.juliadata.org/stable/#Implementing-the-Interface-(i.e.-becoming-a-Tables.jl-source)-1). Most of the popular packages for loading in data from different formats do so already, so you probably won't have to worry about this.
+
+Here, we have a `path` to a csv file, which we'll load in using [CSV.jl](https://github.com/JuliaData/CSV.jl) package, and get a DataFrame using [DataFrames.jl](https://github.com/JuliaData/DataFrames.jl).  
+If your data is present in a different format, you could use a package which supports loading that format, provided that the final object created supports the required interface.
```

Comment:
What is this required interface? Can you link it? Or is this a general comment?

Reply:
Ah, seems like you are referring to the Tables.jl Interface, maybe explicitly note that?

Reply:
Yes, this was referring to the Tables.jl interface. Sure I'll do that.